spec/models/comment_spec.rb

```diff
@@ -0,0 +1,14 @@
+require 'rails_helper'
+
+Rspec.describe Comment, type: :model do
```

Comment:
- [ ] Your tests do not run because of this error. It should be RSpec, not Rspec.

  ![comment-rspec](https://user-images.githubusercontent.com/54848363/190685571-29cd53bd-0e21-463b-93ae-cc2ac6287d15.jpg)